Package.swift

```diff
@@ -8,7 +8,7 @@ let package = Package(
         .library(name: "Tracing", targets: ["Tracing"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/apple/swift-distributed-tracing-baggage.git", from: "0.2.0"),
+        .package(url: "https://github.com/apple/swift-distributed-tracing-baggage.git", .upToNextMinor(from: "0.2.0")),
```

Comment:
I don't think we should do this in a library.

The current issue is just an outcome that we're removing/breaking Apis because all this is pre 1.x, it's not something we need to future guard against.